gulpfile.babel.js

```diff
@@ -23,6 +23,7 @@ const nodeResources = (() => {
   return _.chain(result.toString().trim().split(/\r?\n/))
     // remove app root path
     .tap((r) => r.shift())
+    .filter((dep) => dep.match(/node_modules/g).length == 1)
```

Comment:
@dstreet `npm3` is a flat structure. In order to run the app, we need all dependant packages irrespective of the depth.